docs/organizations/managing-security-and-risk.md

```diff
@@ -104,6 +104,37 @@ The following table defines item priorities and days to fix the underlying Codac
 | Medium                  | Medium        | 90            |
 | Minor                   | Low           | 120           |
 
+## Access management<!-- Rename this. We're not really only managing access, are we? -->
+
+Organization owners can assign the Security Manager role to organization members, as well as revoke it. Security managers share some permissions with organization owners:
+
+|Permission|Organization Owner|Security Manager|
+|---|---|---|
+|Access Security and Risk Management|Yes|Yes|
+|Access originating items (Codacy repositories, Jira issues)|Yes<!-- TODO admin may nothave access here? Confirm -->|Inherits original permissions|
+|Manage integrations|Yes|<!-- TODO confirm if security managers cans manage integrations -->|
+|Assign and revoke the Security Manager role|Yes|No|
+
+To assign the Security Manager role to a user:
+
+1.  Access the **Security and Risk Management configuration** page.
+
+1.  In the Permissions area, use the search field to find the relevant user.
+
+    ![Security and Risk Management access management](images/security-risk-management-access-management.png)
+
+1.  Click the relevant row.<!-- TODO reword this -->
```

Comment:
It's up to you, but I'd probably merge this last (rather obvious) step with the previous one. Something like "[...] find and click the relevant user" perhaps?